package-overrides/github/twbs/bootstrap@4.0.0-beta.json

```diff
@@ -0,0 +1,24 @@
+{
+  "main": "js/bootstrap",
+  "files": null,
+  "ignore": [
+    "dist/js/npm"
+  ],
+  "directories": {
+    "lib": "dist"
```

Comment:
In previous overrides for bootstrap the SASS files were included, for those who work directly with the source, for example by overriding bootstrap mixins.

To preserve this use case it would be best to remove the `"directories"` property and instead qualify `"main"` with `dist/`.